tests/e2e/support/objects/app-files/resource/index.ts

```diff
@@ -82,12 +83,18 @@ export class Resource {
     const startUrl = this.#page.url()
     await moveOrCopyResource({ ...args, page: this.#page, action: 'copy' })
     await this.#page.goto(startUrl)
+    if (!config.ocis) {
+      await this.#page.reload()
+    }
```

Comment:
any issue only in oC10?

Reply:
https://github.com/owncloud/web/issues/8172